### ISSUE TITLE
[Enhancement]: per project config file nvim

### ISSUE THREAD

**Issue by nikolatzotchev** [AdaCore/ada_language_server#1222]
### Enhancement Request

in the wiki for [nvim](https://github.com/AdaCore/ada_language_server?tab=readme-ov-file#integration-with-neovims-built-in-lsp-client) its written: 
> The Ada Language Server's settings are described [here](https://github.com/AdaCore/ada_language_server/blob/master/doc/settings.md). Configuring neovim to use project-specific settings is described neovim's [lspconfig wiki](https://github.com/neovim/nvim-lspconfig/wiki/Project-local-settings)

By following the [link](https://github.com/neovim/nvim-lspconfig/wiki/Project-local-settings#use-native-server-file-configs) you see in `nvim-lspconfig`:

> If you came across an LSP server that doesn't provide a way to configure server via local files, please consider to fill an issue at its repo.

So my question is, is this available when using the `ada_language_server` with nvim? I see that it exists for `VS code` and also `vim`. I will be ideal for me if I could configure settings on project level.

**Comment by eliericha** [comment]
Hello @nikolatzotchev ,

We are already working on this feature to make it easier to use the Ada Language Server with editors. The idea is to make ALS automatically load a `.als.json` file if it exists in the directory where ALS is started. So stand by for this new feature in the next release :)

**Comment by eliericha** [comment]
Hello @nikolatzotchev ,

The [latest ALS release](https://github.com/AdaCore/ada_language_server/releases/tag/26.0.202412191) includes support for automatically loading a `.als.json` file when it exists in the directory where ALS is started (typically the root of a project). Let us know if it works well for you.

**Comment by nikolatzotchev** [comment]
@eliericha yes! Thank you this seems to fix this issue. I have another question that I wanted to ask (and since I do not know where is the correct place, will just do it here). As far as I understand `als` uses `gnatformat` by default, `gnatformat` by default states that it uses [3 space indentation ](https://docs.adacore.com/live/wave/gnatformat/html/user-guide/user-guide/usage.html). So do you know why my als formats my code with 2 spaces :thinking:. Everything else seems to be formatted correctly and as far as I can see there is not setting for this in the `als`. Thank you again for your work!

**Comment by eliericha** [comment]
I suggest opening a new issue to discuss this. Can you tell us which editor you are using in this case? Is it also neovim?
